### PR TITLE
Minor fix to layout algorithm for graphs

### PR DESCRIPTION
**Pull request by sanjaylall**

### CODE REVIEW

File: src/graphlayout.jl [hunk 1]
```diff
@@ -36,6 +36,9 @@ function gradientalg(f, df, epochs, x0)
                 eta = 0.5 * eta
                 theta .+= dtheta
             end
+            if sum(abs.(dtheta)) < 1e-16
+                return theta
```
Comment (cc10512) [inline src/graphlayout.jl:41]: Shouldn't 1e-16 be a symbolic constant? It would make it more clear what kind of threshold are we looking at.

Comment (sanjaylall) [reply src/graphlayout.jl:41]: Good point. It's the minimum stape size.  I added a constant to the gradient algorithm code.

**Review by cc10512 — APPROVED** [review]

File: src/graphlayout.jl (hunk 1 above)
Comment (sanjaylall) [reply src/graphlayout.jl:41]: Oops forgot that const doesn't work for local vars in Julia

**Comment by cc10512** [comment]
I thought you had github actions setup to do CI. Are they setup only for the Callisto.jl repo?

**Pull request merged**